### DISCUSSION TITLE
NotionNext博客的交流群

### DISCUSSION THREAD

**Discussion by wukooc** [tangly1024/NotionNext#419] [Ideas]
希望大佬可以建一个交流群，探讨NotionNext的玩法

**Comment by catmangox** [comment]
我也希望群组建一个。

**Comment by tangly1024** [comment]
感谢提议，我创建了一个telegram群组和微信交流群 ~目前只有我一个人~，欢迎扫码加入：

- Telegram 群组在这： https://t.me/+sOvnlf2QKZQ3NDU1

- 微信群组： 请访问 https://tangly1024.com/about#9bcf6d8f32db4a3eb87612f06b0a0acb 获取加群方式

**Reply by C-Joey** [reply]
最好还是用telegram 我看一些技术交流群都是用tg

**Reply by tangly1024** [reply]
👌，tg群我建一个

**Reply by kitety** [reply]
可以再发一下二维码吗，失效了

**Reply by tangly1024** [reply]
👌 已更新

**Reply by liqun98** [reply]
可以再更新一下二维码嘛

**Comment by tangly1024** [comment]
Telegram 群组在这： https://t.me/+sOvnlf2QKZQ3NDU1
